packages/searchkit-schema/README.md

```diff
@@ -184,4 +184,78 @@ Will provide a GraphQL API where you can perform queries like:
 }
 ```
 
-See [Schema Query Guide](https://searchkit.co/docs/guides/graphql-schema-queries-cheatsheet) for more examples.
\ No newline at end of file
+See [Schema Query Guide](https://searchkit.co/docs/guides/graphql-schema-queries-cheatsheet) for more examples.
+
+### Getting highlights for matches
```

Comment:
could you add this in guides too please? https://searchkit.co/docs/guides/elasticsearch-setup-indexing and also mention it in the api docs https://searchkit.co/docs/reference/schema. These docs are featured in markdown in the docs folder

Reply:
I don't really see how this would fit in https://searchkit.co/docs/guides/elasticsearch-setup-indexing. The highlighting doesn't require any preliminary setup in ES, it is a query time thing as far as I understand. Isn't https://searchkit.co/docs/quick-start/api-setup a better place for discussing this? At the end after https://searchkit.co/docs/quick-start/api-setup#modify-elasticsearch-request-body, maybe.

Reply:
ah what i meant is to create a new page "Field highlighting" in guides section and move the readme contents into that. If you dont know, thats fine, i can do it when I merge.

Reply:
OK, I better let you do it. 😃 You know much better your documentation structure.